### ISSUE TITLE
show baseline exclusion region on specplot

### ISSUE THREAD

**Issue by mpound** [GreenBankObservatory/dysh#55]
figure out x_axis_transform. see my test_spectrum notebook.

**Comment by mpound** [comment]
To explain this cryptic comment.  I have never been a fan of the big rectangles on a plot to indicate where the baseline was  fitted.   The Bell Labs singledish package `comb` (which I worked on decades ago!), had a line underneath the plot that represented the mask used in subtracting a baseline.  In this example it is -5 to 20 km/s.

[comb_plot.pdf](https://github.com/user-attachments/files/19373604/comb_plot.pdf)

I was trying to reproduce this in the test_spectrum notebook a long time ago.

**Comment by etsmit** [comment]
Yes, I remember discussing this. Thanks for the email ping. I was originally going to try reproducing what GBTIDL does (which is a box and whiskers plot without the whiskers for each individual selection region), but maybe I can try doing both and see what people think.

**Comment by mpound** [comment]
@etsmit Is this implemented in the current version of the plotter?
